js/components/frame.js

```diff
@@ -336,7 +337,18 @@ class Frame extends ImmutableComponent {
           this.runOnDomReady()
           delete this.runOnDomReady
         }
+        let zoomCallback = (e) => {
+          if (!e.isMainFrame) {
+            return
+          }
+          this.webview.removeEventListener(e.type, zoomCallback)
+          const zoomPercentage = this.frame && this.frame.get('lastZoomPercentage')
```

Comment:
muon should is supposed to handle this automatically so I think we should fix the problem there. If we need something on the browser-laptop side we should do it in tabs.js because all of the webview methods are deprecated (especially getter methods because they are sync ipc)